File: src/main/java/com/iflat/pam/bean/CommitteeDetail.java
package com.iflat.pam.bean;

import java.util.Date;

/**
 * Created by tyriv on 2016/8/2.
 */
public class CommitteeDetail {
    private String id;
    private String pid;
    private String title;
    private String name;
    private String sex;
    private String degree;  // 文化程度
    private Date birth;
    private String adminTitle;
    private String tel;
    private String creatorAcc;
    private String creatorName;
    private Date createrTime;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getPid() {
        return pid;
    }

    public void setPid(String pid) {
        this.pid = pid;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSex() {
        return sex;
    }

    public void setSex(String sex) {
        this.sex = sex;
    }

    public String getDegree() {
        return degree;
    }

    public void setDegree(String degree) {
        this.degree = degree;
    }

    public Date getBirth() {
        return birth;
    }

    public void setBirth(Date birth) {
        this.birth = birth;
    }

    public String getAdminTitle() {
        return adminTitle;
    }

    public void setAdminTitle(String adminTitle) {
        this.adminTitle = adminTitle;
    }

    public String getTel() {
        return tel;
    }

    public void setTel(String tel) {
        this.tel = tel;
    }

    public String getCreatorAcc() {
        return creatorAcc;
    }

    public void setCreatorAcc(String creatorAcc) {
        this.creatorAcc = creatorAcc;
    }

    public String getCreatorName() {
        return creatorName;
    }

    public void setCreatorName(String creatorName) {
        this.creatorName = creatorName;
    }

    public Date getCreaterTime() {
        return createrTime;
    }

    public void setCreaterTime(Date createrTime) {
        this.createrTime = createrTime;
    }
}

File: web/app/model/pam/CommitteeDetail.js
Ext.define('iFlat.model.pam.CommitteeDetail', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'committeeDetail.id', mapping: 'id', type: 'string'},
        {name: 'committeeDetail.pid', mapping: 'pid', type: 'string'},
        {name: 'committeeDetail.title', mapping: 'title', type: 'string'},
        {name: 'committeeDetail.name', mapping: 'name', type: 'string'},
        {name: 'committeeDetail.sex', mapping: 'sex', type: 'string'},
        {name: 'committeeDetail.degree', mapping: 'degree', type: 'string'},
        {name: 'committeeDetail.birth', mapping: 'birth', type: 'date'},
        {name: 'committeeDetail.adminTitle', mapping: 'adminTitle', type: 'string'},
        {name: 'committeeDetail.tel', mapping: 'tel', type: 'string'},
        {name: 'committeeDetail.creatorAcc', mapping: 'creatorAcc', type: 'string'},
        {name: 'committeeDetail.creatorName', mapping: 'creatorName', type: 'string'},
        {name: 'committeeDetail.createrTime', mapping: 'createrTime', type: 'date'},
    ]
});